### DISCUSSION TITLE
my wallet page says not synced[help]

### DISCUSSION THREAD

**Discussion by ubthemm** [Chia-Network/chia-blockchain#4457] [Support]
**Describe the issue**
PLEASE NOTE!: The issue tracker is NOT a place to seek general support. You are better served starting a conversation in the Discussions area at https://github.com/Chia-Network/chia-blockchain/discussions or in the Keybase.io #support channel. If you create a support request here, it may or may not receive a response before being closed, and we cannot guarantee a timely response, or one at all. In the event that your issue is deemed to be the result of a bug, we will endeavor to review and update it accordingly as such instead of closing it.

**To Reproduce**
Steps that led up to your issue.

**Expected behavior**
A clear and concise description of what you expected to happen before you encountered your issue.

**Screenshots**

**Additional context**

**Comment by lvcivs** [comment]
Do you see any errors in the `debug.log` file?

**Comment by ubthemm** [comment]
İt was wallet no syncing i had to update to new version

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
